tm2py/config.py

```diff
@@ -803,6 +803,7 @@ class HighwayTollsConfig(ConfigItem):
     """
 
     file_path: pathlib.Path = Field()
+    bridetoll_file_path: pathlib.Path = Field()
```

Comment:
@ychtw `bridetoll` - typo?

Reply:
thanks, fixed in c5ffefa26ab05ae27650283586df62fb6a62196c

Reply:
Does not look like it.

Reply:
oops, did not add all changed files. now fixed in b79d69b1ff59e7706c2861e4140633670fd0775d

Reply:
Bring it to `develop_bart`, please.